python/ray/rllib/models/multiagentfcnet.py

```diff
@@ -39,10 +39,9 @@ def _init(self, inputs, num_outputs, options):
 
         # check for a shared model
         shared_model = custom_options.get("multiagent_shared_model", 0)
```

Comment:
Clarification: `shared_model` is meant to be a boolean now? It'd be good to add a comment for the expected type.

Reply:
If it is meant to be a boolean, the parameter can be renamed to `is_shared_model` or similar.

Reply:
Sorry, which clarifying question? I've renamed the parameter